types/src/block/signed_block.rs

```diff
@@ -0,0 +1,48 @@
+use core::fmt::{self, Display, Formatter};
+
+use crate::{Block, BlockSignatures};
+
+/// A block and signatures for that block.
+#[derive(Clone, Debug, PartialEq, Eq)]
+#[cfg_attr(
+    any(feature = "std", feature = "json-schema", test),
+    derive(serde::Serialize, serde::Deserialize)
```

Comment:
We tend to import these traits so we don't need to fully qualify them here.

Reply:
done